### PR TITLE
Ad Posting Functionality

### PR DESCRIPTION
**Pull request by SiamakSamie**
"Add want" Functionality
Posts collection

### CODE REVIEW

**Review by JKukulsky — APPROVED** [review]
Ran locally, outputs as predicted. Changes ok with me.

File: soen341webapp/client/main.html [hunk 1]
```diff
@@ -1,59 +1,128 @@
...
-                                <a href="#">I will make this</a>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </ul>
+
+{{> search }}
+
+
```
Comment (DarrinFong) [inline soen341webapp/client/main.html:15]: Descriptions can be added to the dynamic elements to increase readability.

File: soen341webapp/client/main.html [hunk 2]
```diff
@@ -1,59 +1,128 @@
...
+        </div>
+        </div>
+
+
+          
+          
+       {{/each}}
+      </ul>
+  </div>
+
```
Comment (DarrinFong) [inline soen341webapp/client/main.html:46]: This file remains to be formated

File: soen341webapp/client/main.js [hunk 3]
```diff
@@ -1,12 +1,49 @@
...
 Template.body.helpers({
-    ads(){
-        return Ads.find({}); // ads is thee name of collection
+    posts(){
+        return Posts.find({}); 
     }
+});
 
+
+
```
Comment (DarrinFong) [inline soen341webapp/client/main.js:21]: Short description of every function will be useful for future references

File: soen341webapp/client/main.js [hunk 4]
```diff
@@ -1,12 +1,49 @@
...
+           desc,
+           createdAt: new Date()
+        });
+        
+        //clear form
+        event.target.reset();
+        
+        //close modal
+        $('.modal').modal('close');
+
```
Comment (DarrinFong) [inline soen341webapp/client/main.js:45]: return of bool is never used

**Review by DarrinFong — APPROVED** [review]
Code works, additional items should be aligned horizontally before vertically. CSS is to be reviewed and changed to implement this change

**Pull request merged**